File: Latest_pay_2.0/Mini_Project_N/src/app/payment-response.ts
export class PaymentResponse {
    orderId: string;
    paymentId: string;
    paymentLinkId: string;
    paymentStatus: string;
    paymentSignature: string;
  
    constructor(
      orderId:string ,
      paymentId: string,
      paymentLink: string,
      paymentStatus: string,
      paymentsignature: string
    ) {
      this.orderId = orderId;
      this.paymentId = paymentId;
      this.paymentLinkId = paymentLink;
      this.paymentStatus = paymentStatus;
      this.paymentSignature = paymentsignature;
    }
  }
  

File: Latest_pay_2.0/paymentIntegration/src/main/java/com/payment/paymentIntegration/entity/PaymentResponse.java
package com.payment.paymentIntegration.entity;


import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.*;
import org.springframework.boot.autoconfigure.web.WebProperties;

@Data
@Entity
@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class PaymentResponse {
    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    private String id;
    private Long orderId;
    private String paymentId;
    private String paymentLinkId;
    private String paymentStatus;
    private String paymentSignature;

}
